infrastructure/lambda/template.yaml

```diff
@@ -397,16 +439,21 @@ Resources:
                 - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${AWS::StackName}/DCS/JWE/EncryptionCertForPassportToEncrypt"
                 - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${AWS::StackName}/DCS/JWE/SigningCertForPassportToVerify"
                 - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${AWS::StackName}/DCS/JWE/EncryptionKeyForPassportToDecrypt"
+                - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${AWS::StackName}/HMPODVAD/API/GraphQl/QueryString"
```

Comment:
Have we documented in confluence anywhere what all these params are and what there for?

Reply:
I did something similar here when these last got complicated https://govukverify.atlassian.net/wiki/spaces/PYI/pages/3567485402/DCS+Certificate+Catalogue